src/components/common/trees/index.js

```diff
@@ -6,34 +6,47 @@ const Trees = ({ trees }) => {
     <div className="trees">
       {trees.number_of_trees_to_offset === 0 ? (
         <p id="niceEmoji" className="emojis">
-          {trees.number_of_trees_to_offset}
+        CO<sub>2</sub> emitted = {trees.number_of_trees_to_offset} trees burned
           <img src={niceEmoji} alt="nice emoji" />
         </p>
       ) : trees.number_of_trees_to_offset >= 600 ? (
         <p className="emojis">
-          {trees.number_of_trees_to_offset}
+        CO<sub>2</sub> emitted = {trees.number_of_trees_to_offset} trees burned
           <span role="img" aria-label="string">
             🏭😷
           </span>
         </p>
       ) : trees.number_of_trees_to_offset >= 475 ? (
         <p className="emojis">
-          {trees.number_of_trees_to_offset}
+        CO<sub>2</sub> emitted = {trees.number_of_trees_to_offset} trees burned
           <span role="img" aria-label="string">
             🙁
           </span>
         </p>
       )
-      : trees.number_of_trees_to_offset >= 350 ? (
+      : trees.number_of_trees_to_offset >= 100 ? (
         <p className="emojis">
-          {trees.number_of_trees_to_offset}
+          CO<sub>2</sub> emitted 
+          <br></br>=<br></br>
+          {trees.number_of_trees_to_offset} trees burned
+          <br></br>
           <span role="img" aria-label="string">
-            🙂
+            😐
           </span>
         </p>
+      )  : trees.number_of_trees_to_offset >= 0 ? (
+        <p className="emojis">
+          CO<sub>2</sub> emitted 
+          <br></br>=<br></br>
+          {trees.number_of_trees_to_offset} trees burned
+          <br></br>
+          <span role="img" aria-label="string">
+            🙂
+          </span>
+        </p>     
```

Comment:
Here you repeat the same bit of code a couple of times. I think it would be better if we moved some of the functionality into a different function.
For example, instead of creating a bunch of cases where only the emoji is changed, just make it call a function that returns the relevant emoji. Something like:
```
function getEmojiForNumTrees(number_of_trees_to_offset) {
  if (number_of_trees_to_offset >= 600) {
    return "🏭😷";
  } else if (number_of_trees_to_offset >= 475) {
    return "🙁";
  } else if (number_of_trees_to_offset >= 100) {
    return "😐";
  } else if (number_of_trees_to_offset >= 0) {
    return "🙂";
  } else {
    return "😀";
  }
}
```
Then just have a special case for 0.